packages/ra-ui-materialui/src/input/ReferenceArrayInput.tsx

```diff
@@ -78,14 +78,23 @@ import { AutocompleteArrayInput } from './AutocompleteArrayInput';
  * a `setFilters` function. You can call this function to filter the results.
  */
 export const ReferenceArrayInput = (props: ReferenceArrayInputProps) => {
-    const { children, reference } = props;
+    const {
+        children = defaultChildren,
+        reference,
+        sort = { field: 'id', order: 'DESC' },
```

Comment:
As these are already the default values in `useReferenceArrayInputController`, you can omit them.